server/index.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express')
+const app = express()
+
+const port = 5000
+app.listen(port, () => {console.log("running on port", port)})
+
+app.get('/', (req, res) => {

```

Comment:
Typically, the convention for JS when dealing with unused variables is to use an underscore.
Also, let's just name the route to something like 'test' instead of the home directory.
```
app.get('/', (_, res) => ...
```